content/references-rest-api.md

```diff
@@ -147,6 +147,38 @@ Example response:
 }
 ```
 
+## Users
+
+### GET users
+
+Users endpoint can be used to retrieve users' ids who are subscribed to a specific presence channel.
```

Comment:
I would suggest to say "[...] list of user ids which belongs to users who has an active subscription to [...]"